files/jupyterhub/Dockerfile

```diff
@@ -18,7 +18,8 @@ RUN wget https://get.docker.com -q -O /tmp/getdocker && \
     chmod +x /tmp/getdocker && \
     sh /tmp/getdocker && \
     rm /tmp/getdocker && \
-    apt-get -y clean && apt-get -y autoremove
+    apt-get -y clean && apt-get -y autoremove && \
+    apt-get install iputils-ping
```

Comment:
I'd add this into line 21 so maybe line 21 changes to:

```
apt-get -y install iputils-ping && \
    apt-get -y clean && \
    apt-get -y autoremove
```

The Travis build failed because the `-y` flag needs to be included in the install command to automatically tell the installer that yes, you want this to install non-interactively.